cmd/argo/commands/stop.go

```diff
@@ -33,29 +58,55 @@ func NewStopCommand() *cobra.Command {
   argo stop @latest
 `,
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 && !o.isList() {
```

Comment:
it should be ok to have no args, e.g. if you were to do

```
kubectl get wf -o name | xargs argo stop
```

That should be fine, even if kubectl returned zero workflows.

What happens with both list options AND args? Should that be an error? 

Is this consistent with `argo list` and `argo delete`?

Reply:
When we use the `xargs` command it works fine, because the `xargs` command generates and executes lines. Indeed, `kubectl get wf -o name | xargs argo stop` works, but `argo stop` returns the help message.

At now it works only with the list option when we use both list options and args, and it is not consistent with `argo delete`, so I need to fix it. But in my opinion, Argo should block using both because it makes the user confused. In `kubectl` it returns the error when we use both list options and args like the below.

```shell
k get pods -l selector=true foo
error: name cannot be provided when a selector is specified
```

Reply:
I've fixed it to be consistent with the `delete` & `list` command.